
# this packs the sequence the robot identity an the time stamp
Header header

# Navigation data of aria robots
string	 Mode
string	 Status
float64  Voltage
float64  X
float64  Y
float64  Th
float64  Vel
float64  RotVel
float64  LatVel
float64  Temperature

#add localization score information